cypress/objects/timeLogging.js

```diff
@@ -0,0 +1,38 @@
+class TimeLogging {
+    getUrl() {
+        return cy.url()
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title')
+    }
+
+    getCalendar() {
+        return cy.get('.calendar')
+    }
+
+    getTitleForm() {
+        return cy.get('.tile.form')
+    }
+
+    getUserInfoTitles() {
```

Comment:
What is a 'user info title'? What hides behind this class is the name of the user, right? So why not call it something like `getUserName()`? :)